recipes/metaknowledge/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "metaknowledge" %}
+{% set version = "3.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f137dce7ed7e1fb3a73730716d37e2b5757d9fc94f3bb50f8e82338c38b17dae
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+  host:
```

Comment:
Please drop the build and also add the same constraint to python in host

Reply:
Thanks for the comments! It's done.